src/foundry/foundry.js/clientDocuments/combat.d.ts

```diff
@@ -16,7 +18,202 @@ declare global {
      * The configuration setting used to record Combat preferences
      */
     static CONFIG_SETTING: 'combatTrackerConfig';
+
+    /** Track the sorted turn order of this combat encounter */
+    turns: InstanceType<ConfiguredDocumentClass<typeof Combatant>>[];
+
+    /** Record the current round, turn, and tokenId to understand changes in the encounter state */
+    protected current: RoundData;
+
+    /** Track the previous round, turn, and tokenId to understand changes in the encounter state */
+    protected previous: RoundData;
+
+    /**
+     * Track whether a sound notification is currently being played to avoid double-dipping
+     * @defaultValue `false`
+     */
+    protected _soundPlaying: boolean;
+
+    /** Get the Combatant who has the current turn. */
+    get combatant(): InstanceType<ConfiguredDocumentClass<typeof Combatant>>;
+
+    /** The numeric round of the Combat encounter */
+    get round(): number;
+
+    /**
+     * A reference to the Scene document within which this Combat encounter occurs.
+     * If a specific Scene is not set in the Combat Data, the currently viewed scene is assumed instead.
+     */
+    get scene(): InstanceType<ConfiguredDocumentClass<typeof Scene>>;
+
+    /** Return the object of settings which modify the Combat Tracker behavior */
+    get settings(): ClientSettings.Values[typeof Combat['CONFIG_SETTING']];
+
+    /** Has this combat encounter been started? */
+    get started(): boolean;
+
+    /** The numeric turn of the combat round in the Combat encounter */
+    get turn(): number;
+
+    get visible(): true;
+
+    /**
+     * Set the current Combat encounter as active within the Scene.
+     * Deactivate all other Combat encounters within the viewed Scene and set this one as active
+     */
+    activate(): Promise<this>;
```

Comment:
```suggestion
    activate(): Promise<InstanceType<ConfiguredDocumentClass<typeof Combat>>[]>;
```

(foundries JSDoc is incorrect, consider creating an issue)